mysql-plugin/widgets/Mysql-batchsource.json

```diff
@@ -95,6 +95,14 @@
           "label": "Database",
           "name": "database"
         },
+        {
+          "widget-type": "connection-browser",
+          "widget-category": "plugin",
```

Comment:
Can you just add this to all the plugins we migrated? Should be same code change except the type

Reply:
done , I was planning to make the changes in other PR :)